status-im/status-react
# Onboarding - Enable biometrics screen

fixes #15029 

### Summary

Enable biometrics screen

<img src="https://user-images.githubusercontent.com/18485527/226937169-bb03353b-2523-4950-85c9-db06e56748c6.png" width="300">

#### Platforms

- Android
- iOS

#### Areas that maybe impacted

##### Functional

- new account

### Steps to test

- Open Status
- Start onboarding flow
- Reach to Enable biometrics step
- Verify it is working as expected

status: ready
